shared/src/commonMain/kotlin/me/saket/press/shared/editor/EditorPresenter.kt

```diff
@@ -196,8 +203,10 @@ class EditorPresenter(
       .distinctUntilChanged()
       .map(folderPaths::isArchived)
       .distinctUntilChanged()
+      .combineLatestWith(syncer.status())
+      .distinctUntilChanged()
```

Comment:
@saket While this seems to perfectly work, please let me know if you know anything better than this in this situation.